test/relations.test.js

```diff
@@ -4987,7 +4987,7 @@ describe('relations', function() {
       Category.findById(category.id, function(err, cat) {
         if (err) return done(err);
         var link = cat.items.at(0);
-        link.updateAttributes({notes: 'Updated notes...'}, function(err, link) {
+        cat.items.updateById(link.id, {notes: 'Updated notes...'}, function(err, link) {
```

Comment:
`Model1.Model2.updateById` is one of the operations of embedsMany, and it is giving the same behavior but with a workaround for the issue we have with the failing connectors. The purpose was just to update and that's why the next failing is depending on this one.